source/MapPanel.h

```diff
@@ -89,8 +89,12 @@ class MapPanel : public Panel {
 	virtual bool Drag(double dx, double dy) override;
 	virtual bool Scroll(double dx, double dy) override;
 
+	double MinColor() const;
```

Comment:
comment? only if you want to

Reply:
I added them for the variables but will move them here

Reply:
```suggestion
	// Minimum and maximum color ranges used by mapColor for the legend of the custom outfitter prices.
	double MinColor() const;
```